dataeng/jobs/createJobs.groovy

```diff
@@ -59,7 +60,7 @@ try {
 }
 
 def taskMap = [
-    ANALYTICS_EMAIL_OPTIN_JOB: AnalyticsEmailOptinJob,
+/*    ANALYTICS_EMAIL_OPTIN_JOB: AnalyticsEmailOptinJob,
```

Comment:
I assume the commenting out of this is so you could just deploy the new job?

Reply:
Correct.